### PR TITLE
Azure arm64 upi e2e destructive test config

### PR DESCRIPTION
**Pull request by aleskandro**
This PR adds the Prow configs for the qe e2e destructive tests and the upgrade tests against Azure UPI on ARM64 clusters.

Refers [ARMOCP-489](https://issues.redhat.com//browse/ARMOCP-489)

### CODE REVIEW

**Comment by aleskandro** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-4.13-upgrade-from-stable-4.12-azure-upi-p3-f28

**Comment by aleskandro** [comment]
/pj-reharse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-azure-upi-p3-f28-destructive

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml [hunk 1]
```diff
@@ -24152,6 +24233,88 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 38 23 10 * *
```
Comment (MayXuQQ) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:24238]: in https://github.com/openshift/release/blob/771b675615346f57f2190c5736b5abb92f1a56ce/ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml#L12605
periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-upi-p3-f28-destructive
cron: 38 23 1 * *
How about change to 38 23 16 * * ?

**Comment by MayXuQQ** [comment]
Deploying 04_bootstrap
ERROR: {"status":"Failed","error":{"code":"DeploymentFailed","message":"At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.","details":[{"code":"BadRequest","message":"{\r\n  \"error\": {\r\n    \"code\": \"BadRequest\",\r\n    \"message\": \"The selected VM size 'Standard_D4ps_v5' cannot boot Hypervisor Generation '1'. If this was a Create operation please check that the Hypervisor Generation of the Image matches the Hypervisor Generation of the selected VM Size. If this was an Update operation please select a Hypervisor Generation '1' VM Size. For more information, see https://aka.ms/azuregen2vm\"\r\n  }\r\n}"}]}}
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:79","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2023-01-30T07:52:04Z"}
error: failed to execute wrapped command: exit status 1 
INFO[2023-01-30T07:52:11Z] Step azure-upi-p3-f28-upi-install-azure failed after 8m0s.

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml (hunk 1 above)
Comment (aleskandro) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:24157]: I changed it properly to distribute the executione time between the archs. We'll get amd64 executing on 1st of each month, the arm64 one on 16th.

**Comment by aleskandro** [comment]
> Deploying 04_bootstrap ERROR: 

As anticipated, upgrades profiles are currently available only starting from a stable release. Changed the PR description and scope to not cover upgrades for now (until we get a first RC).

**Comment by aleskandro** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-azure-upi-p3-f28-destructive

**Comment by aleskandro** [comment]
Tests for the upgrade will be added at the end of this sprint, when ec3 will be available

**Comment by aleskandro** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-azure-upi-p3-f28-destructive

**Comment by aleskandro** [comment]
installation succeeded. 

/unhold

/assign @jhou1 

can you help merging this?

**Pull request merged**